Dockerfile

```
@@ -23,6 +23,7 @@ COPY . .
 RUN cargo build --release
 
 FROM base AS tester
+COPY . .
 COPY --from=builder /app/target/release/fennel-node /app/fennel-node
 RUN cargo test --features=runtime-benchmarks
```

This is just meant to run tests while Docker is building. We don't need to save this layer or copy anything over, that's my mistake. This'll just get trashed when the final image is produced.

```suggestion
FROM base AS tester
RUN cargo test --features=runtime-benchmarks
```

---

ok. noted.